### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by nahylh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,7 +176,9 @@ Taxis could be a challenge during raining days.
 
 ### MRT and Buses
 
-The public transport system runs on a reloadable contactless card called EZ-Link. There are 1, 2, and 3 day unlimited "Tourist" passes. There is no need to buy these unless you plan on making 4+ trips per day.
+The public transport system runs on a reloadable contactless card called EZ-Link. There are 1, 2, and 3 day unlimited "Tourist" passesThere is no need to buy these unless you plan on making 4+ trips per day. Available also are standard tickets which can be re-used for up to 6 times, purchasable at General Ticketing Machines (GTM) at all LRT and MRT stations.
```
Comment (winston) [inline README.md]: Missing a fullstop + space between `passesThere`.

**Comment by winston** [comment]
Other than the one change above, everything else is awesome. Will merge once updated.

Thank you!!! :heart: :bow:

**Comment by nahylh** [comment]
Added a bit more information and made the necessary changes! :)

**Comment by winston** [comment]
Thank you!!!

**Pull request merged**